### PR TITLE
Improvements to redocly format

### PR DESCRIPTION
**Pull request by farisdurrani**
# Improvements to redocly format

**What user problem are we solving?**
Grouping and making valid paths. See https://fluffy-rattlesnake-92.redoc.ly for Redocly preview docs/

Screenshot 2023-06-17 at 13 54 39

**Testing Methodology**
Spun up Redocly API preview docs

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+openapi: 3.1.0
+info:
+  version: 1.0.0
+  title: Deep Learning Playground API
+  termsOfService: https://example.com/terms/
+  contact:
+    email: contact@example.com
+    url: http://example.com/contact
+  license:
```
Comment (karkir0003) [inline openapi/openapi.yaml:8]: incorrect. we are mit licensed?

File: openapi/openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    description: Everything about s3
+  - name: test
+    description: Test API endpoints
+  - name: trainspace
+    description: Everything about trainspace
+servers:
+  - url: https://example.com/api/v1
+paths:
+  /dataset/defaultDataset:
+    $ref: paths/dataset/defaultDataset.yaml
```
Comment (karkir0003) [inline openapi/openapi.yaml:27]: do tags get linked to yaml files? @farisdurrani

Comment (farisdurrani) [reply openapi/openapi.yaml:27]: Yup, we'd have to put the tag manually inside each yaml file

File: openapi/openapi.yaml (hunk 1 above)
Comment (farisdurrani) [reply openapi/openapi.yaml:8]: Fixed

**Review by karkir0003 — APPROVED** [review]

**Pull request merged**